deps/Makefile

```diff
@@ -1185,6 +1185,7 @@ OPENBLAS_BUILD_OPTS += NO_AVX2=1
 endif
 
 $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/config.status: $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/Makefile
+	cd $(dir $@) && patch -p1 < ../openblas-make.patch
 ifeq ($(OS),WINNT)
 	cd $(dir $@) && patch -p1 < $(SRCDIR)/openblas-win64.patch
```

Comment:
you'll have to delete this patch, and update the checksums if we're going to update to 0.2.17